core/toolbox.js

```diff
@@ -233,25 +233,24 @@ Blockly.Toolbox.prototype.position = function() {
   var svgPosition = goog.style.getPageOffset(svg);
   var svgSize = Blockly.svgSize(svg);
   if (this.horizontalLayout_) {
-    treeDiv.style.left = svgPosition.x + 'px';
+    treeDiv.style.left = '0px';
```

Comment:
These should just be 0, in accordance with the Google style guide:
https://google.github.io/styleguide/htmlcssguide.xml?showone=0_and_Units#0_and_Units